### PR TITLE
Add `addModifier` and `removeModifier` plugin wrappers. :rocket:

### PR DESCRIPTION
**Pull request by brunoskonrad**
- addModifier just add a modifier if it hasn't;
- removeModiifer remove a modifier if it exist;
- add a `npm run setup` to setup development environment;

### CODE REVIEW

File: lib/jQuery.toggleModifier.js [hunk 1]
```diff
@@ -11,39 +11,68 @@
 }(function($) {
   'use strict';
 
-  $.fn.toggleModifier = function(modifier) {
-    var selectorReturn = $(this),
-        self = this;
-
+  /**
```
Comment (viniciusalmeida) [inline lib/jQuery.toggleModifier.js]: For what exactly do you need of this comment? This helps you with something?

File: tests/acceptance/general_behavior.js [hunk 2]
```diff
@@ -42,3 +42,39 @@ QUnit.test('Remove the correct modifier when the element contains two or more',
   assert.ok(this.subject.hasClass('block__element--another-modifier'),
     'The modifier not toggled remains equal');
 });
+
+QUnit.test('Add a modifier', function(assert) {
+  this.subject.addClass('block__element');
+  this.subject.addModifier('modifier');
+
+  assert.ok(this.subject.hasClass('block__element--modifier'),
+    'The modifier not toggled remains equal');
```
Comment (viniciusalmeida) [inline tests/acceptance/general_behavior.js]: What do you think about improve this assertion descriptions?

**Comment by brunoskonrad** [comment]
I've made the changes that you have pointed. It's ok, @viniciusalmeida? :smile:

**Comment by viniciusalmeida** [comment]
Done @brunoskonrad, thanks!

**Comment by brunoskonrad** [comment]
Nosostros! :metal:

**Comment by viniciusalmeida** [comment]
Was released in [this version](https://github.com/viniciusalmeida/jQuery.toggleModifier/releases/tag/0.2.0)

**Pull request merged**